### DISCUSSION TITLE
"0.20mm_QUALITY_SV06_Ellis.ini" does appear as one of PrusaSlicer's "Print Settings" options

### DISCUSSION THREAD

**Discussion by travmo** [bassamanator/Sovol-SV06-firmware#101] [General]
**Describe the bug**
When importing the PrusaSlicer Config Bundle, it only shows 4 of the 5 print profiles in PrusaSlicer. 

The file "0.20mm_QUALITY_SV06_Ellis.ini" does appear in the PrusaSlicer's "print" folder, however it does not appear in the UI as an option you can select. I reviewed the profile and cannot determine why it does not appears as an option, as expected. 

**To Reproduce**
1. Go to PrusaSlicer application (in my case, on macOS)
2. Click on File > Import > Import Config Bundle
3. Scroll down to file "PrusaSlicer_config_bundle-SV06-Both.ini" that was downloaded 
4. Under the Plater tab, review Print Settings options. 
5. Observe how 4 of the 5 print configurations appear.

**Expected behavior**
I am hoping to utilize that fifth profile as needed, and while I could manually create a profile based on the settings, I imagine others are facing this issue as well. 

**Screenshots**

<img width="1275" alt="Screenshot 2023-10-27 at 9 53 33 AM" src="https://github.com/bassamanator/Sovol-SV06-firmware/assets/14857257/a1a6432f-3c6b-43a4-bd32-6b200092207b">

Note: I added underscores (_) to the name to see if possibly the spaces were part of the issue. The issue occurs with and without underscores as seen in my screenshot

**What printer are you using?**
- SV06

**Is your local copy of this configuration up to date?**
- Yes it is

**PrusaSlicer Version**
Version 2.6.1+MacOS-arm64

**Comment by bassamanator** [comment]
This is more of a 'how do I make this happen in PS' rather than an issue with anything. 

You'll likely have to change the dependencies section for your other custom print setting to link it to the printer profile included with my bundle:
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/61985779/cd40bef4-86a4-49b5-8f2a-da052e2fb4ac). I would check the PS docs, should be easy to figure out.

In any case, you should see your missing print settings when you select your custom printer (the one that is not included in the bundle from this repo).

**Reply by travmo** [reply]
Ah, that lead me down the right path, thanks bassamanator! I discovered that the other 4 profiles were configured as:

`compatible_printers_condition =`

While the missing profile shows:

`compatible_printers_condition = printer_notes=~/.*PRINTER_VENDOR_PRUSA3D.*/ and printer_notes=~/.*PRINTER_MODEL_MK3.*/ and nozzle_diameter[0]==0.4`

Removing everything after the = (to match the others) allowed it to be seen.

**Reply by Heiko910** [reply]
> Ah, that lead me down the right path, thanks bassamanator! I discovered that the other 4 profiles were configured as:
> 
> `compatible_printers_condition =`
> 
> While the missing profile shows:
> 
> `compatible_printers_condition = printer_notes=~/.*PRINTER_VENDOR_PRUSA3D.*/ and printer_notes=~/.*PRINTER_MODEL_MK3.*/ and nozzle_diameter[0]==0.4`
> 
> Removing everything after the = (to match the others) allowed it to be seen.

Hi Travmo,
Where did you find the individual profiles? I want to make the 5th profile visible as well?
Thanks
